### ISSUE TITLE
Loss and mAP curves are not plotted on google colab

### ISSUE THREAD

**Issue by oubaid2** [AlexeyAB/darknet#7361]
Used darknet pre-trained weights with YOLOv3 on a custom data set. I trained the model on max_batches of 4000 for a 1 class object detection, but no graph for loss or neither mAP showed in my drive folder as it is supposed to do based on what i read. 
My configuration file:
[net]/ # Testing/ # batch=1 /# subdivisions=1 /# Training/batch= 64 /subdivisions=16 /width=416 /height=416 /channels=3 /momentum=0.9 /decay=0.0005 /angle=0 /saturation = 1.5 /exposure = 1.5 / hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 4000
policy=steps
steps=3600,4400
scales=.1,.1
This is the training command i used :

`!darknet/darknet detector train custom_data/labelled_data.data darknet/cfg/yolov3_custom.cfg custom_weight/darknet53.conv.74 -dont_show -map`
This how it looks from my side:
![image](https://user-images.githubusercontent.com/58915263/107501601-e8820180-6b97-11eb-9eca-6c1f05255fb4.png)

The chart.png is not there, not in any folder.
![image](https://user-images.githubusercontent.com/58915263/107500590-a1474100-6b96-11eb-8ec3-1db3805dbe45.png)

I also tried this command:
`!darknet/darknet detector train custom_data/labelled_data.data darknet/cfg/yolov3_custom.cfg custom_weight/darknet53.conv.74 -dont_show -mjpeg_port 8090 -map`  Followed the instructions on the repostery.
==> Website Page does not open (chrome/firefox).

What possibly did go wrong with my situation? This is really frustrating..

**Comment by stephanecharette** [comment]
How did you build it?  The chart requires OpenCV and `OPENCV=1` in the makefile.

(Also, from your screenshot you'll want to look at bad.list.)

**Comment by oubaid2** [comment]
@stephanecharette this is my makefile:
GPU=1
CUDNN=1
OPENCV=1
OPENMP=0
DEBUG=0

 andthis badl.list

data/labels/32_0.png
data/labels/33_0.png
data/labels/34_0.png
data/labels/35_0.png
data/labels/36_0.png
data/labels/37_0.png
data/labels/38_0.png
data/labels/39_0.png
data/labels/40_0.png
data/labels/41_0.png
data/labels/42_0.png
data/labels/43_0.png
data/labels/44_0.png
data/labels/45_0.png
data/labels/46_0.png
data/labels/47_0.png
data/labels/48_0.png
data/labels/49_0.png
data/labels/50_0.png
data/labels/51_0.png
data/labels/52_0.png
data/labels/53_0.png
data/labels/54_0.png
data/labels/55_0.png
data/labels/56_0.png
data/labels/57_0.png
....
..

These file paths are in the darknet folder, "yolov3/darknet/data/...png".
 I don't know why it keep reading them

**Comment by MKiremitci** [comment]
#7075 maybe this issue can help the other people

**Comment by RamaRam99** [comment]
> Used darknet pre-trained weights with YOLOv3 on a custom data set. I trained the model on max_batches of 4000 for a 1 class object detection, but no graph for loss or neither mAP showed in my drive folder as it is supposed to do based on what i read.
> My configuration file:
> [net]/ # Testing/ # batch=1 /# subdivisions=1 /# Training/batch= 64 /subdivisions=16 /width=416 /height=416 /channels=3 /momentum=0.9 /decay=0.0005 /angle=0 /saturation = 1.5 /exposure = 1.5 / hue=.1
> 
> learning_rate=0.001
> burn_in=1000
> max_batches = 4000
> policy=steps
> steps=3600,4400
> scales=.1,.1
> This is the training command i used :
> 
> `!darknet/darknet detector train custom_data/labelled_data.data darknet/cfg/yolov3_custom.cfg custom_weight/darknet53.conv.74 -dont_show -map`
> This how it looks from my side:
> ![image](https://user-images.githubusercontent.com/58915263/107501601-e8820180-6b97-11eb-9eca-6c1f05255fb4.png)
> 
> The chart.png is not there, not in any folder.
> ![image](https://user-images.githubusercontent.com/58915263/107500590-a1474100-6b96-11eb-8ec3-1db3805dbe45.png)
> 
> I also tried this command:
> `!darknet/darknet detector train custom_data/labelled_data.data darknet/cfg/yolov3_custom.cfg custom_weight/darknet53.conv.74 -dont_show -mjpeg_port 8090 -map` Followed the instructions on the repository using this
> link `http://ip-address:8090` and i used `!curl ipecho.net/plain` to determine my ip-adress.
> ==> Website Page does not open (chrome/firefox).
> 
> What possibly did go wrong with my situation? This is really frustrating..

The chart.png doesn't save in the backup directory(aka ur google drive). It normally just saves in the /darknet directory and not in any of its folders but down below among the single files and since i assume colab sorts the files alphabetically it will be quick to find.

However in ur case
`!darknet/darknet detector train custom_data/labelled_data.data darknet/cfg/yolov3_custom.cfg custom_weight/darknet53.conv.74 -dont_show -map`
According to this code i assume it saved in the darknet folder in the darknet directory.
